docs/introduction/what_is_renku.rst

```diff
@@ -0,0 +1,63 @@
+.. _what_is_renku_verbose:
+
+What is Renku?
+==============
+
+The Renku Project is a web platform (:ref:`renkulab`) and a command-line
+interface (:ref:`renku`) built on top of open source components for researchers,
+data scientists, educators, and students to help manage:
+
+* code,
+* data,
+* execution environments, and
+* workflows
+
+Renku combines many widely-used open-source tools in order to make any project
```

Comment:
**suggestion**: Renku combines many widely-used open-source tools to equip every project on the
platform with resources that aid reproducibility, repeatability, reusability and
shareability.